docs/user-guide/api-mediation/api-mediation-multi-tenancy.md

```diff
@@ -0,0 +1,246 @@
+
+## Use the /registry endpoint in Cloud Gateway
+
+The /registry endpoint provides information about services onboarded to all domain Gateways and the Central Gateway.
+
+### Configuration
+
+The /registry endpoint is disabled by default. Use the following environment variable to enable this feature:
+
+`APIML_CLOUDGATEWAY_REGISTRY_ENABLED=TRUE`
+
+### Authentication
+The /registry endpoint is authenticated by the client certificate. Cloud Gateway accepts certificates that are trusted. The user name is obtained from the common name of the client certificate.
+
+You will receive 401 when not properly authenticated.
+
+### Authorization
+Only the users configured by the following environment variable are allowed to use the /registry endpoint.
+
+`APIML_SECURITY_X509_REGISTRY_ALLOWEDUSERS=USER1,user2,User3`
+
+This parameter allows the setting of multiple users as a comma-separated list.
+
+You will receive 403 when not properly authorized.
+
+### Request
+
+GET /cloud-gateway/api/v1/registry                   List services in all domains
+GET /cloud-gateway/api/v1/registry/{apimlId}    List services in apimlId domain
+
+### Response
+
+Example:
+
+```
+[
+    {
+        "apimlId": "apiml1",
+        "services": [
+            {
+                "status": "UP",
+                "customMetadata": {},
+                "apiId": [
+        "zowe.apiml.gateway"
+],
+                "serviceId": "gateway"
+            }
+        ]
+    },
+    {
+        "apimlId": "apiml2",
+        "services": [
+            {
+                "status": "UP",
+                "customMetadata": {},
+                "apiId": [
+        "zowe.apiml.gateway"
+],
+                "serviceId": "gateway"
+            }
+        ]
+]
+```
+
+## Validate successful configuration
+
+Use the /registry endpoint to validate the successful configuration. The response should contain all API ML domains represented by apimlId and information about onboarded services.
+
+## Troubleshooting
+
+- ZWESG100W Cannot receive information about services on API Gateway with apimlId 'Gateway-CA32' because: Received fatal alert: certificate_unknown; nested exception is javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknown
+  - The trust between the domain and the Cloud Gateway was not established. Review your certificate configuration.
+
+- No debug messages similar to Gateway-CA32 completed with onComplete are produced. 
+  - Domain Gateway is not correctly onboarded to Discovery Service in Central API ML. Review Gateway static definition. Check the Central Discovery Service dashboard if the domain Gateway is displayed. 
+
+## Onboarding domain cloud-gateway service to central discovery service
+
+The central Cloud Gateway can onboard all domain’s cloud-gateways. This can be achieved by similar to gateway’s additional registrations:
+
```

Comment:
Please list what the option are to onboard domain cloud-gateway services to central discover services.